File: Chapter13/Lesson2/Exercise1/Partial/CS/MathLibrary/Math.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace MathLibrary
{
    public class Math
    {
        private int _value = 0;

        public int Value
        {
            get
            {
                return _value;
            }
            set
            {
                _value = value;
            }
        }

        public void Add(int n)
        {
            _value += n;
        }

        public void Subtract(int n)
        {
            _value -= n;
        }
    }
}

File: Chapter13/Lesson2/Exercise1/Partial/VB/MathLibrary/Math.vb
Public Class Math
    Private _value As Integer = 0

    Public Property Value() As Integer
        Get
            Return _value
        End Get
        Set(ByVal value As Integer)
            _value = value
        End Set
    End Property

    Public Sub Add(ByVal n As Integer)
        _value += n
    End Sub

    Public Sub Subtract(ByVal n As Integer)
        _value -= n
    End Sub
End Class

